clang/lib/Serialization/ASTWriter.cpp

```diff
@@ -3262,17 +3262,18 @@ void ASTWriter::WritePragmaDiagnosticMappings(const DiagnosticsEngine &Diag,
 /// Write the representation of a type to the AST stream.
 void ASTWriter::WriteType(QualType T) {
   TypeIdx &IdxRef = TypeIdxs[T];
-  if (IdxRef.getIndex() == 0) // we haven't seen this type before.
+  if (IdxRef.getValue() == 0) // we haven't seen this type before.
```

Comment:
NIT: For the purpose of improving readability, I wonder if there is a better name than `getValue`? Would something like `getSerializedValue()` or `getGlobalIndex()` be appropriate here?

Reply:
I feel it might not be needed. Since the type `TypeIdx` shows the information that this is index from serialization to me.

Reply:
SG, let's keep as is